crates/driver/openapi.yml

```diff
@@ -28,7 +28,7 @@ paths:
             - `sell`: amount is in sell_token, out_amount is in buy_token
           schema:
             type: string
-            enum: ["buy", "sell"]
+            enum: [ "buy", "sell" ]
```

Comment:
I noticed many of these unrelated openapi changes in the past.
Can we keep the change of this PR only related to the affected endpoints and instead add a linting step for these files as a github action so there is not back and forth on different styles in the future?